classes/class-uabb-init.php

```diff
@@ -53,6 +53,8 @@ public function __construct() {
 			add_action( 'admin_notices', array( $this, 'admin_notices' ) );
 			add_action( 'network_admin_notices', array( $this, 'admin_notices' ) );
 		}
+		// Hook the load_plugin_textdomain function to the init action.
+		add_action( 'init', array( $this, 'load_plugin_textdomain' ) );
```

Comment:
we are calling same function 2 times in this file, I hope this is intentional

Reply:
Okay, I'll keep the load_plugin_textdomain call in the includes method and removing the redundant hook from the constructor. @sushmak02